src/eos-updater-poll-common.c

```diff
@@ -76,7 +76,9 @@ G_STATIC_ASSERT (G_N_ELEMENTS (order_key_str) == EOS_UPDATER_DOWNLOAD_LAST + 1);
 static const gchar *const EOS_UPDATER_BRANCH_SELECTED = "99f48aac-b5a0-426d-95f4-18af7d081c4e";
 #endif
 
-static const gchar *const REFSPEC_TO_UPGRADE_ON_PATH = PKGDATADIR "/upgrade-refspec";
+/* This path looks hardcoded, but it is actually a relative path, relative to
+ * the current OSTree deployment and not the rootfs. */
```

Comment:
What’s the reasoning for this change? If we decide to change the `PREFIX` for eos-updater, surely we want it to look in a correspondingly different subpath of the deployment?